pkg/cloudprovider/cloudapi/aws/aws_crd_helpers.go

```diff
@@ -24,21 +24,31 @@ import (
 
 	runtimev1alpha1 "antrea.io/nephe/apis/runtime/v1alpha1"
 	cloudcommon "antrea.io/nephe/pkg/cloudprovider/cloudapi/common"
+	"antrea.io/nephe/pkg/cloudprovider/securitygroup"
 	"antrea.io/nephe/pkg/labels"
+	"antrea.io/nephe/pkg/util/k8s"
 )
 
 const ResourceNameTagKey = "Name"
 
 // ec2InstanceToInternalVirtualMachineObject converts ec2 instance to VirtualMachine runtime object.
 func ec2InstanceToInternalVirtualMachineObject(instance *ec2.Instance, vpcs map[string]*ec2.Vpc, namespace string,
 	account *types.NamespacedName, region string) *runtimev1alpha1.VirtualMachine {
-	tags := make(map[string]string)
-	vmTags := instance.Tags
-	if len(vmTags) > 0 {
-		for _, tag := range vmTags {
-			tags[*tag.Key] = *tag.Value
+	vmTags := make(map[string]string)
+	if len(instance.Tags) > 0 {
+		for _, tag := range instance.Tags {
+			key := *tag.Key
+			value := *tag.Value
+			// skip any tags added by nephe for internal processing
```

Comment:
nit: add a period . at the end of comment.

Reply:
done